src/software/new_geom/geom_constants.h

```diff
@@ -0,0 +1,13 @@
+namespace GeomConstants
+{
+    // Due to internal representation of doubles being slightly less accurate/consistent
+    // with some numbers and operations, we consider points that are very close together
+    // to be equal (since they likely are, just possibly slightly misrepresented by the
+    // system/compiler). We use this EPSILON as a threshold for comparison. 1e-15 was
+    // chosen as a value because doubles have about 16 consistent significant figures.
+    // Comparing numbers with 15 significant figures gives us a
+    // small buffer while remaining as accurate as possible.
+    // http://www.cplusplus.com/forum/beginner/95128/
+    static constexpr double EPSILON = 1e-15;
+    static constexpr double NEAR    = 1e-9;
```

Comment:
I'm OK with epsilon since it has a meaning derived from an actual limitation of the computer system, but `NEAR` seems way too vague and subjective to me.

Reply:
Should we leave NEAR as `1e-9` in it's usages?

Reply:
Seems like it's only used to check if a Point is near another point (ie. they're very close but not equal). I don't know if we use `isClose` without any parameters, so maybe this variable and the `isClose()` function can be removed?

Reply:
Yeah I was gonna say that we should probably reconsider any functions that use `NEAR`. I'd say remove `isClose()` and see if we can do without.

Reply:
Removed `isClose()` and `NEAR` in the latest update